chemprop/cli/utils/parsing.py

```diff
@@ -53,7 +53,9 @@ def parse_csv(
             column
             for column in df.columns
             if column
-            not in set(input_cols + (ignore_cols or []) + (splits_col or []) + (weight_col or []))
+            not in set(  # if splits or weight is None, df.columns will never have None
```

Comment:
do we want to keep the comment here?

Reply:
Yes, I think leave the comment, especially since this is in the CLI and not core code. A confused user could come back to the PR to see why we coded it this way (using a list of [None] as the default), but maybe easier to just let them know inline that we knew what we were doing.